src/appshell/view/dockwindow/dockpageview.cpp

```diff
@@ -213,6 +232,130 @@ DockPanelView* DockPageView::findPanelForTab(const DockPanelView* tab) const
     return !panels.isEmpty() ? panels.first() : nullptr;
 }
 
+void DockPageView::reorderSections()
+{
+    //! NOTE: In some cases, such as setting visible true,
+    //! it is necessary to give the UI time to render the content, so we will add a delay.
+    QTimer::singleShot(2000, this, [this](){
```

Comment:
We already use interval: 500 (StyledDialogView.qml) for the same purpose. It might be worth declaring one interval somewhere and reusing it twice

Reply:
Unfortunately, an interval of 500ms is not suitable for the current problem since it's too short
Also, an interval of 2000ms is not suitable for the dialog because it's too long to start voicing accessibility information